packages/camera/camera/test/camera_test.dart

```diff
@@ -59,9 +59,15 @@ void main() {
     test('debugCheckIsDisposed should not throw assertion error when disposed',
         () {
       const MockCameraDescription description = MockCameraDescription();
-      final CameraController controller = CameraController(
+      final CameraController controller = CameraController.withSettings(
         description,
-        ResolutionPreset.low,
+        mediaSettings: const MediaSettings(
+          resolutionPreset: ResolutionPreset.low,
+          fps: 15,
+          videoBitrate: 200000,
+          audioBitrate: 32000,
+          enableAudio: true,
```

Comment:
Same question for all the existing tests in this file. If these changes are necessary, then we can't ship this API as-is.

Reply:
not necessary, removed